Readme.md

```diff
@@ -109,6 +109,17 @@ log.info('this will be written to /tmp/debug.stream.out and /tmp/info.stream.out
 log.fatal('this will be written to /tmp/debug.stream.out, /tmp/info.stream.out and /tmp/fatal.stream.out')
 ```
 
+### pinoms.level set accessor
+
+You can set the level to _all streams_ by changing the level property.
+It accepts the same parameters of pino.
```

Comment:
yes I know. I was just pointing it out that I'm not sure of the implementation of this behavior :).